src/features/contactverzoek/AfdelingSearch.vue

```diff
@@ -0,0 +1,82 @@
+<template>
```

Comment:
svp uitzoeken of afdelingsearch en groepsearch generieker opgezet kunnen worden. hebben nu veel overlap

Reply:
Done. Medewerker Search zou evt ook meegenomen kunnen worden maar daar gaan in de toekomst mogelijk ook nog dingen in veranderen.